### PR TITLE
Add skip inactivity to the recording player

### PR DESCRIPTION
**Pull request by rcmarron**
## Changes

This PR adds the concept of "inactive sections" to the recording player. It
* Shows inactive segments to the user
* Adds a toggle that allows users to skip inactive segments
* If a segment is being skipped, it overlays a "Skipping inactivity" text

image

## How did you test this code?

Verified that it:
* Skips segments when the player hits an inactive segment
* Skips segments if you scrub to the middle of an inactive segment
* Skips segments if the player starts on an inactive one
* Does not skip segments if the user toggles off "skip inactivity"
* Starts skipping a segment if the user toggles on "skip inactivity" in the middle of a segment

Note: We **really** should write tests for `SessionRecordingsPlayerLogic.ts`. Going to put up a PR for that soon.

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/Seekbar.tsx [hunk 1]
```diff
@@ -54,6 +54,19 @@ export function Seekbar(): JSX.Element {
...
         <div className="rrweb-controller-slider">
             <div className="slider" ref={sliderRef} onMouseDown={handleDown} onTouchStart={handleDown}>
+                <div className="inactivity-bar">
+                    {sessionPlayerData?.metadata?.segments?.map((segment: RecordingSegment) => (
+                        <div
+                            key={`${segment.windowId}-${segment.startTimeEpochMs}`}
+                            className={clsx('activity-section', !segment.isActive && 'inactive-section')}
+                            style={{
+                                width: `${
+                                    (100 * segment.durationMs) / sessionPlayerData.metadata.recordingDurationMs
```
Comment (mariusandra) [inline frontend/src/scenes/session-recordings/player/Seekbar.tsx:64]: Will this `/ sessionPlayerData.metadata.recordingDurationMs` properly take into account the multiple windows? Doesn't that add more stuff to the bar that the nr of seconds?

File: frontend/src/scenes/session-recordings/player/PlayerController.tsx [hunk 2]
```diff
@@ -49,6 +49,10 @@ export function PlayerController(): JSX.Element {
                     ))}
                 </Select.OptGroup>
             </Select>
+            <div className="rrweb-inactivity-toggle">
+                <span className="inactivity-label">Skip inactivity</span>
+                <Switch checked={skipInactivitySetting} onChange={setSkipInactivitySetting} size="small" />
+            </div>
```
Comment (mariusandra) [inline frontend/src/scenes/session-recordings/player/PlayerController.tsx:55]: The "skip inactivity" label could also be a click trigger for the switch.

**Review by mariusandra — APPROVED** [review]
Looks good. 👍 

Seems to work well locally with multiple windows as well, but just in case asked one inline question about it.


Plus one question about the switch. The rest is good to go!

File: frontend/src/scenes/session-recordings/player/Seekbar.tsx (hunk 1 above)
Comment (rcmarron) [reply frontend/src/scenes/session-recordings/player/Seekbar.tsx:64]: Good question, `recordingDurationMs` is calculated based on the segments not just subtracting the start time from the end time, so it accounts for segments that overlap and when segments aren't continuous.

File: frontend/src/scenes/session-recordings/player/PlayerController.tsx (hunk 2 above)
Comment (rcmarron) [reply frontend/src/scenes/session-recordings/player/PlayerController.tsx:55]: Good call - making the change now.

**Pull request merged**